frontend/src/styles/modules/_main-header.scss

```diff
@@ -5,15 +5,16 @@
 
     .wordmark {
       @include respond-to('not-small') {
-        @include hidpi-background-image('wordmark', 296px 64px);
+        @include hidpi-background-image('wordmark', 302px 64px);
         height: 64px;
-        width: 296px;
+        width: 302px;
       }
-      @include hidpi-background-image('wordmark', 190px 41px);
+      @include hidpi-background-image('wordmark', 194px 41px);
+      background-repeat: none;
```

Comment:
Seems weird that the logo seems to be still showing up again in the lower resolution(mobile) view when `background-repeat` is set here.. hmmm